### PR TITLE
README: add AWS EMR instructions

### PR DESCRIPTION
**Pull request by guybortnikov**
## Description
Add AWS EMR Installation instructions to project README

## Motivation and Context
Make gProfiler installation easy and well explained on AWS EMR.

### CODE REVIEW

**Comment by guybortnikov** [comment]
Comment to include artwork
image (10)

File: README.md [hunk 1]
```diff
@@ -268,6 +268,38 @@ gcloud dataproc clusters create <CLUSTER NAME> \
...
+     - Expand *Bootstrap Actions* section
+     - Add Action of type *Custom Action*
+     - Fill in script location with the appropriate script location on your S3, for example `s3://my-s3-bucket/grofiler-bootstrap.sh`
+     
+   - AWS CLI Example: 
+     ```bash
+     aws emr create-cluster --name MY-Cluster ... --bootstrap-actions "Path=s3://my-s3-bucket/grofiler-bootstrap.sh"
+     ```
+     
+
```
Comment (Jongy) [inline README.md:301]: Too many blank lines

**Comment by Jongy** [comment]
Missing the image?

**Comment by guybortnikov** [comment]
image
Add new image

File: README.md [hunk 2]
```diff
@@ -268,6 +268,36 @@ gcloud dataproc clusters create <CLUSTER NAME> \
   - Replace `<CLUSTER NAME>` with the cluster name you wish to use
   - Replace `<REGION>` with the region you wish to use
 
+
+## Running on AWS EMR
```
Comment (Jongy) [inline README.md:272]: This is placed in between 2 sections of dataproc (the next heading, `### Debugging problems`, belongs to dataproc)

Comment (guybortnikov) [reply README.md:286]: Good catch, fixed

**Review by Jongy — APPROVED** [review]

**Pull request merged**